src/Groups/homomorphisms.jl

```diff
@@ -612,7 +612,7 @@ function isomorphism(::Type{T}, G::GAPGroup) where T <: Union{SubPcGroup, SubFPG
      f = fun(GapObj(G))::GapObj
      @req f !== GAP.Globals.fail "Could not convert group into a group of type $T"
      H = T(GAPWrap.ImagesSource(f))
-# TODO: remove the next line once GAP 4.13.0 is available in Oscar
+# TODO: remove the next line once GAP 4.13.0 is available in Oscar -> still broken in GAP 4.14.0
      GAP.Globals.UseIsomorphismRelation(GapObj(G), GapObj(H))
```

Comment:
at least for the case of PermGroups. After I remove this line as indicated in the comment:
```julia
julia> G = dihedral_group(6)
Pc group of order 6

julia> iso = isomorphism(PermGroup, G)
Group homomorphism
  from pc group of order 6
  to permutation group of degree 3 and order 6

julia> has_order(codomain(iso))
false
```
With the line still in, the last result is `true` instead.

This got caught by the doctest https://github.com/oscar-system/Oscar.jl/blob/07e803543ba2dea53825026d16dfc68cfab8a4f8/src/Groups/homomorphisms.jl#L598 that no longer printed the order of the codomain.